pkgs/tools/security/fprintd/tod.nix

```diff
@@ -18,4 +19,29 @@
       rev = "v${version}";
       sha256 = "sha256-rOTVThHOY/Q2IIu2RGiv26UE2V/JFfWWnfKZQfKl5Mg=";
     };
+
+    patches = patches ++ [
+      (fetchpatch {
+        name = "use-more-idiomatic-correct-embedded-shell-scripting";
+        url = "https://gitlab.freedesktop.org/libfprint/fprintd/-/commit/f4256533d1ffdc203c3f8c6ee42e8dcde470a93f.patch";
+        sha256 = "sha256-4uPrYEgJyXU4zx2V3gwKKLaD6ty0wylSriHlvKvOhek=";
+      })
+      (fetchpatch {
+        name = "remove-pointless-copying-of-files-into-build-directory";
+        url = "https://gitlab.freedesktop.org/libfprint/fprintd/-/commit/2c34cef5ef2004d8479475db5523c572eb409a6b.patch";
+        sha256 = "sha256-2pZBbMF1xjoDKn/jCAIldbeR2JNEVduXB8bqUrj2Ih4=";
+      })
+      (fetchpatch {
+        name = "build-Do-not-use-positional-arguments-in-i18n.merge_file";
+        url = "https://gitlab.freedesktop.org/libfprint/fprintd/-/commit/50943b1bd4f18d103c35233f0446ce7a31d1817e.patch";
+        sha256 = "sha256-ANkAq6fr0VRjkS0ckvf/ddVB2mH4b2uJRTI4H8vPPes=";
+      })
+    ];
+
+    postPatch = ''
+      ${postPatch}
```

Comment:
This has the potential of breaking when the trailing newline of `oldAttrs.postPatch` is removed.

> also where is meta?

I don't know, but I didn't remove it.

Reply:
It is safe to assume that postPatch will keep the final new line.

> I don't know, but I didn't remove it.

Then we should add one to keep meta in good shape. Or maybe we could inherit it?

Reply:
I added `inherit (oldAttrs) meta`, although I don't expect this to change anything. After all `fprintd-tod` is some kind of overlay over `fprintd`. Do you think I should make this more fine-grained?

Reply:
Turns out that `inherit (oldAttrs) meta` is completely useless because it obviously doesn't update `meta.position`. I've added some metadata and myself as maintainer.